pykern/pkconst.py

```diff
@@ -31,3 +33,33 @@
 
 #: Copied from numconv, which copied from RFC1924
 BASE62_CHARS = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz'
+TWO_PI = 2 * math.pi
+RT_TWO_PI = math.sqrt(2*math.pi)
+RT_2_OVER_PI = math.sqrt(2/math.pi)
+pi = math.pi
+RT_2 = math.sqrt(2.)
+TWO_PI = 2 * math.pi
+RT_TWO_PI = math.sqrt(2*math.pi)
+RT_2_OVER_PI = math.sqrt(2/math.pi)
+c = scipy.constants.c                  # speed of light [m/s]
```

Comment:
We do not use side comments. They do not show up in Sphinx/Napoleon. They are hard to manage, because they are dependent on the font and the width of the strings. If names change, they are misaligned. Our coding style avoids misalignments when names change.